stdlib/3/curses/textpad.pyi

```diff
@@ -0,0 +1,11 @@
+from _curses import _CursesWindow
+from typing import Callable, Union
+
+def rectangle(win: _CursesWindow, uly: int, ulx: int, lry: int, lrx: int) -> None: ...
+
+class Textbox:
+    stripspaces: bool
+    def __init__(self, w: _CursesWindow) -> None: ...
+    def edit(self, validator: Callable[[int], int]) -> str: ...
```

Comment:
The argument is called `validate` and is both optional and Optional (i.e., it should be `validate: Optional[Callable[[int], int]] = ...`).

Reply:
Ah, documentation said ```edit([validator])``` both make sense but source code always wins.